sampledb/src/main/java/com/rean/controller/DbApp.java

```diff
@@ -0,0 +1,13 @@
+package com.rean.controller;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class DbApp {
```

Comment:
As a next step for this PoC lets containerise this demo application and launch it in AWS ECS.
Please take this up as you next task and demo it.